src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Transformation non prise en charge dans la configuration du contenu. 
+i18nReady: true 
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+--- 
+
+import DontEditWarning from '~/components/DontEditWarning.astro'
```

Comment:
```suggestion
```

I believe you can safely remove this import, as the page is being translated and so doesn't need the component anymore.